components/Renderers/Writing.tsx

```
@@ -20,13 +22,20 @@ const Writing = ({ fileUrl, description }: WritingProps) => {
+  const isOverflowed = height > 206;
   return (
-    <div
-      className="font-spectral bg-grey-moss-100 p-2 rounded-md text-md !normal-case text-left"
-      dangerouslySetInnerHTML={{
-        __html: text.replaceAll("\n", "<br/>"),
-      }}
-    />
+    <>
+      <div
+        className="font-spectral bg-grey-eggshell p-2 text-md !normal-case text-left"
+        dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)